content/docs/studio/user-guide/account-management.md

```diff
@@ -82,10 +82,140 @@ In this section, you can,
 
 ## Cloud credentials
 
-Here, you can see all the credentials you have added for your cloud / data
+Here, you can see all the credentials you have added for your cloud and data
 remotes. You can also add new credentials as well as edit or delete existing
-credentials. The credentials that you add here will become available to use with
-your projects in Iterative Studio.
+credentials. These credentials useful in two cases:
+
+- To
+  [fetch project data from data remotes](/doc/studio/user-guide/projects-and-experiments/configure-a-project#data-remotes--cloud-storage-credentials)
+- To
+  [create cloud resources for running experiments](/doc/studio/user-guide/projects-and-experiments/run-experiments#cloud-experiments)
+
+To add credentials, click on your user icon in the top right corner and go to
+your [Profile]. Navigate to `Cloud Credentials` and click `Add credentials`.
+Select the cloud provider (note that
+[cloud experiments](/doc/studio/user-guide/projects-and-experiments/run-experiments#cloud-experiments)
+currently support AWS and GCP). Depending on the provider, you will be asked for
+more details.
+
+![](https://static.iterative.ai/img/studio/s3_remote_settings_v2.png)
+
+The credentials must have the required permissions. For
+[cloud experiments](/doc/studio/user-guide/projects-and-experiments/run-experiments#cloud-experiments),
+the following permissions are needed:
```

Comment:
Should we also link to https://dvc.org/doc/user-guide/data-management/remote-storage for data remotes?

Reply:
[Added](https://github.com/iterative/dvc.org/pull/4589/commits/e6fdd788a9c837b30f7efe349f7083a01fa823b9).